nodejs/aws-serverless/dynamodb/table.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as aws from "@pulumi/aws";
+import { dynamodb, lambda } from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+import { RunError } from "@pulumi/pulumi";
+import { createLambdaFunction, Handler } from "./../function";
+import { EventSubscription } from "./../subscription";
+
+/**
+ * Arguments to control the event rule subscription.  Currently empty, but still defined in case of
+ * future need.
+ */
+export interface TableEventSubscriptionArgs {
+    /**
+     * The largest number of records that Lambda will retrieve from your event source at the time of invocation. Defaults to `100` for DynamoDB.
+     */
+    readonly batchSize?: number;
+
+    /**
+     * The position in the stream where AWS Lambda should start reading. Must be one of either `TRIM_HORIZON` or `LATEST`.
+     */
+    readonly startingPosition: "TRIM_HORIZON" | "LATEST";
```

Comment:
note that this is required.  given that there is a required arg here, it seems likely that we shoudl require 'args' to be provided in the onXXX mehtods below.  @lukehoban do you concur?

Reply:
Decision:  yes, this is required. so users have to supply it.

Reply:
Resolution: Yes.  This is required.